ramls/isbn.raml

```diff
@@ -0,0 +1,104 @@
+#%RAML 1.0
+title: Inventory API
+version: v1
+protocols: [ HTTP, HTTPS ]
+baseUri: http://localhost
+
+documentation:
+  - title: "ISBN API"
+    content: <b>API for validation and conversion of ISBN-10 and ISBN-13 numbers </b>
+
+types:
+  errors: !include raml-util/schemas/errors.schema
+  isbn:
```

Comment:
Why are these types defined inline, rather than as JSON schema?

@dcrossleyau Do our RAML validation tools support definitions like this?

Reply:
The tools should handle it. And yes, looking at the CI job report from the last run, there were no complaints.

If it is going to be re-used in other places, then better to be an "include".